pillar/edx/sandbox.sls

```diff
@@ -35,7 +35,7 @@ edx:
     ################################################################################
     #################### Forum Settings ############################################
     ################################################################################
-    FORUM_API_KEY: __vault__:gen_if_missing:secret-residential/global/forum-api-key>data>value
+    FORUM_API_KEY: __vault__:gen_if_missing:secret-mitxpro/global/forum-api-key>data>value
```

Comment:
We should pull the business unit from the env data for this so that we don't have to hardcode it per env.